boa/interpret.py

```diff
@@ -159,7 +159,7 @@ def get_compiler_data():
             _ = ret.bytecode, ret.bytecode_runtime
         return ret
 
-    assert isinstance(deployer, type)
+    assert isinstance(deployer, type) or deployer is None
```

Comment:
This should be a vyper deployer by default, why should it be none?

Reply:
this is a bugfix. the deployer can be none according to the function signature.